### PR TITLE
Fix BoundsControl incorrectly calculates world space center of bounds

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
This PR fixes the currently incorrect calculation of the center of bounds in world space.

## Changes
- Fixes: #9366

### CODE REVIEW

File: Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs [hunk 1]
```diff
@@ -784,7 +784,7 @@ private void Update()
                 // also only use proximity effect if nothing is being dragged or grabbed
                 if (!wireframeOnly && currentPointer == null)
                 {
-                    proximityEffect.UpdateScaling(Vector3.Scale(TargetBounds.center, TargetBounds.gameObject.transform.lossyScale) + Target.transform.position, currentBoundsExtents);
+                    proximityEffect.UpdateScaling(TargetBounds.transform.TransformPoint(TargetBounds.center), currentBoundsExtents);
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs:787]: We no longer user `Target.transform` for anything here. Are there any cases where that might be a problem?

Comment (MaxWang-MS) [reply Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs:787]: Could you elaborate a little bit more? I replaced Target with TargetBounds so there is no problem right?

Comment (keveleigh) [reply Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs:787]: I think I'm not sure the difference between `Target` and `TargetBounds` and wondering if ignoring `Target` now might cause us to ignore some positional offset or something

Comment (keveleigh) [reply Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs:787]: Resolved offline. The proximity effect really only cares about the target's collider (`TargetBounds`), regardless of the GameObject's position itself

**Review by keveleigh — APPROVED** [review]

**Pull request merged**